utest.h

```diff
@@ -50,6 +50,7 @@
 #endif
 
 #if defined(_MSC_VER)
+#include <windows.h>
```

Comment:
If you only need this on older MSC_VER's, could you guard the include so that its not used on 2019? It's a big and complicated header.